src/components/scenario/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+
+import { useDispatch, useSelector } from '@/hooks/common/useCustomRedux.ts';
+
+import Button from '@/components/common/button/button';
+import * as S from '@/components/scenario/buttonGroup/buttonGroup.style';
```

Comment:
@/components/scenario/ButtonGroup/ButtonGroup.style 으로 수정하셔야할 것 같습니다!
```suggestion
import * as S from '@/components/scenario/ButtonGroup/ButtonGroup.style';
```